frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -23,6 +24,7 @@ function GameStatistics({ finalScores, setIsStatisticsDisplayed, handleResetGame
         <div className="table-item">score</div>
         <div className="table-item">{finalScores.white}</div>
       </div>
+      {heatmap && <Heatmap heatmap={heatmap} />}
```

Comment:
This always returns true since empty array is always a true value. What I would like to see as a user is a loading info when I wait for the heatmap to load.

Reply:
Done. Info about loading is now displayed when component loads.